client/state/reader/streams/reducer.js

```diff
@@ -73,7 +72,7 @@ export const items = ( state = [], action ) => {
 			}, state );
 
 			// Find any x-posts
-			newXPosts = filter( streamItems, ( postKey ) => postKey.xPostMetadata );
+			newXPosts = streamItems.filter( ( postKey ) => postKey.xPostMetadata );
```

Comment:
Above we needed optional chaining, but here we don't. Is that intentional?

Reply:
Confirming that this comment is no longer relevant, we don't need optional chaining here because we added the `isArray` check above.